lock-api/src/main/java/com/palantir/lock/v2/NamespacedTimelockRpcClient.java

```diff
@@ -20,22 +20,8 @@
  * Adapter that mirrors {@code TimelockRpcClient}, but for convenience automatically provides the namespace as a
  * parameter.
  */
```

Comment:
Possibly shouldn't include this exact javadoc both on the interface and the default implementation?